produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
+
+      if merchant_exists? merchant_identifier
+        UI.success("[DevCenter] Merchant '#{merchant_identifier})' already exists, nothing to do on the Dev Center")
+      else
+        merchant_name = Produce.config[:merchant_name] || merchant_identifier.split(".").map(&:capitalize).reverse.join(' ')
+
+        UI.success("Creating new merchant '#{merchant_name}' with identifier '#{merchant_identifier}' on the Apple Dev Center")
+
+        merchant = Spaceship.merchant.create!(bundle_id: merchant_identifier,
+                                            name: merchant_name,
+                                            mac: Produce.config[:platform].to_s == 'mac')
+
+        if merchant.name != merchant_name
+          UI.important("Your merchant name includes non-ASCII characters, which are not supported by the Apple Developer Portal.")
```

Comment:
Sorry I'm not sure, this is the same as https://github.com/fastlane/fastlane/blob/master/produce/lib/produce/group.rb#L25

I have changed the language slightly in my latest commits to instead read "Your merchant name **might** include..."